addOns/ascanrules/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- GetForPost scan rule uses ComparableResponse to compare post and get responses.
```

Comment:
> GET for POST scan now uses a different comparison mechanism which should be more performant.

Reply:
For this rule it's not about performance (if something it could be less), it's to handle unrelated modifications in the body (e.g. timestamps).

Reply:
I'll tweak the note.

Reply:
Ill change the note. Please let me know if there are other changes so that I can do it at once.

Reply:
Still outstanding